rust-toolchain.toml

```diff
@@ -1,5 +0,0 @@
-[toolchain]
-channel = "1.63.0"
```

Comment:
It was enough to remove just that line, rest of this file should work with `rustup`.

Reply:
Sure, but `rustup override set $MSRV` overrides the toolchain file completely, so it won't do anything (for the CI) anyway.

And since the toolchain file was only there for CI to begin with I figured I would just remove it.